### PR TITLE
Fix: No successful Toast for Sync Survey

### LINKED ISSUE #1489: No successful Toast for Sync Survey
**Summary:** 

On successful completion of the sync survey, no toast notification is raised.

**Steps to reproduce:** 

Open side drawer and click on settings> click on sync survey

**Expected behaviour:** 

It should generate a toast on the successful completion of the sync survey.

**Observed behaviour:** 

On successful completion of the sync survey, no toast notification is raised.

**Device and Android version:** 

Android Version 9.0
Realme 5

**Screenshots:** 

settings

### PR DESCRIPTION
**Pull request by robustTechie**
Fixes #1489 

Please Add Screenshots If there are any UI changes.

toastinsurvey

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java [hunk 1]
```diff
@@ -206,7 +206,8 @@ public void showNetworkIsNotAvailable() {
     @Override
     public void showSurveysSyncSuccessfully() {
         btnCancel.setVisibility(View.INVISIBLE);
-        btnHide.setText(getResources().getString(R.string.dialog_action_ok));
+        dismissDialog();
+        Toast.makeText(getActivity(), "Sucessfully Synced", Toast.LENGTH_SHORT).show();
```
Comment (jawidMuhammadi) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java:210]: Use dynamic string, please :)

Comment (robustTechie) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java:210]: Ok

Comment (AbhilashG97) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java:210]: @robustTechie, please don't use hardcoded text, can you please update your pull request? (^_^)

Comment (robustTechie) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java]: Done

Comment (robustTechie) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java]: > @robustTechie, please don't use hardcoded text, can you please update your pull request? (^_^)

Done

**Pull request merged**